reviveme/models/__init__.py

```diff
@@ -1,5 +1,3 @@
-from .comment import Comment
-from .thread import Thread
```

Comment:
For some reason, these lines cause a circular dependency when I was trying to import ThreadVote and CommentVote in the thread and comment model files. I think we should move away from doing this in general.